Snippets/Core/Core_6/Core_6.csproj

```diff
@@ -316,6 +316,14 @@
     <Content Include="Audit\Audit.config.xml">
       <SubType>Designer</SubType>
     </Content>
+    <Content Include="Azure\Persistence\AzureStorage\AzurePersistence.config.xml">
```

Comment:
It might be a left over from Simon's breaking changes, if it wasn't intentional I can move it around, no problem but wanted to check first :)